shared/telemetry/noop_telemetry_module.go

```diff
@@ -0,0 +1,107 @@
+package telemetry
+
+import (
+	"log"
+
+	"github.com/pokt-network/pocket/shared/config"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	_ modules.TelemetryModule   = &NoopTelemetryModule{}
+	_ modules.EventMetricsAgent = &NoopTelemetryModule{}
+	_ modules.TimeSeriesAgent   = &NoopTelemetryModule{}
+)
+
+type NoopTelemetryModule struct {
+	bus modules.Bus
+}
+
+func NOOP() {
+	log.Printf("\n[telemetry=noop]\n")
+}
+
+func CreateNoopTelemetryModule(cfg *config.Config) (*NoopTelemetryModule, error) {
+	return &NoopTelemetryModule{}, nil
+}
+
+func (m *NoopTelemetryModule) Start() error {
+	NOOP()
+	return nil
+}
+
+func (m *NoopTelemetryModule) Stop() error {
+	NOOP()
+	return nil
+}
+
+func (m *NoopTelemetryModule) SetBus(bus modules.Bus) {
+	m.bus = bus
+}
+
+func (m *NoopTelemetryModule) GetBus() modules.Bus {
+	if m.bus == nil {
+		log.Fatalf("PocketBus is not initialized")
+	}
+	return m.bus
+}
+
+func (m *NoopTelemetryModule) GetEventMetricsAgent() modules.EventMetricsAgent {
+	return m
+}
+
+func (m *NoopTelemetryModule) EmitEvent(namespace, event_name string, labels ...any) {
+
+}
+
+func (m *NoopTelemetryModule) GetTimeSeriesAgent() modules.TimeSeriesAgent {
+	return m
+}
+
+func (p *NoopTelemetryModule) CounterRegister(name string, description string) { NOOP() }
+
+func (p *NoopTelemetryModule) CounterIncrement(name string) { NOOP() }
+
+func (p *NoopTelemetryModule) GaugeRegister(name string, description string) {
+	NOOP()
+}
+
+// Set sets the Gauge to an arbitrary value.
```

Comment:
You kept the comments for some functions but not others.

1. Stay consistent.
2. It's confusing to have these comments in place for the NOOP module.